File: LivePriceBackend/DTOs/Base/BaseAuditableDto.cs
namespace LivePriceBackend.DTOs.Base;

public abstract class BaseAuditableDto
{
    public int Id { get; set; }
    public DateTime? CreatedAt { get; set; }
    public int? CreatedById { get; set; }
    public DateTime? UpdatedAt { get; set; }
    public int? UpdatedById { get; set; }
} 

File: live_price_frontend/lib/core/models/base_auditable_dto.dart
import 'base_dto.dart';

abstract class BaseAuditableDto extends BaseDto {
  final DateTime? createdAt;
  final int? createdById;
  final DateTime? updatedAt;
  final int? updatedById;

  BaseAuditableDto({
    required super.id,
    this.createdAt,
    this.createdById,
    this.updatedAt,
    this.updatedById,
  });
}
